bundles/org.openhab.binding.fronius/src/main/resources/OH-INF/update/update.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+
+	<thing-type uid="fronius:powerinverter">
+		<instruction-set targetVersion="1">
```

Comment:
In case you would like for existing things an update of label and description for the deprecated channel, you need to provide upgrade instructions, this time setting the new label and the new description.